x-pack/plugin/esql/src/main/java/generated/org/elasticsearch/xpack/esql/expression/function/scalar/string/LocateNoStartEvaluator.java

```diff
@@ -0,0 +1,139 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Did this file sneaked in somehow? Seems unrelated

Reply:
Hmm this seems to be some build bug actually ... the file keeps coming up again and again for me, deleting it :)